lib/index.js

```diff
@@ -39,11 +40,19 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   const accountPath = basePath + '/:account';
   const {baseUri} = bedrock.config.server;
 
+  /**
+   * @function createNewAccount
+   * @param {String} path '/'
+   * @param {Function} auth middleware
+   * @param {Function} validator middleware
+   * @param {Function} handler
+   * @description
+   */
   app.post(
     basePath,
     optionallyAuthenticated,
-    // TODO: validate('services.account.postAccounts')
-    asyncHandler(async (req, res) => {
+    validate('account.postAccounts'),
+    asyncHandler(async function createNewAccount(req, res) {
```

Comment:
For consistency purposes and developer friendliness, I prefer the arrow function pattern in our express routes.